content/deploy/centos.yaml

```diff
@@ -13,7 +13,11 @@ chapters:
 
               To deploy Couchbase Mobile to production you must first get familiar with Couchbase Server. While it's certainly possible to run Sync Gateway in isolation for quick and agile development and testing, any production deployment needs Couchbase Server as the back-end. Luckily, connecting the two is incredibly easy. Even more luckily, both Couchbase Server and Sync Gateway are designed to be able to scale out easily and with zero downtime, meaning that we can start with a very small deployment and scale it up later to something that's production ready.
               Couchbase Server can deployed on a whole host of [operating systems](http://www.couchbase.com/nosql-databases/downloads), but for this workshop we'll focus on using CentOS 7.
-
+              
+              ### Pre-requisites
+              
+              - Vagrant 5.0.10 (download it [here](https://www.virtualbox.org/wiki/Download_Old_Builds_5_0_pre18))
```

Comment:
Actually, was going to update this today. I believe it just needs to be pre-5.2, so 5.1 is fine. Will double check

Reply:
Sounds good, I think we need the pre-requisites section either way. Once you know the minimum required version to install.